documentation/whatsnew/fusetools/fusetools-news-10.2.0.AM1.adoc

```diff
@@ -0,0 +1,19 @@
+= Fuse Tooling - What's New in 10.2
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 10.2.0.AM1
+:page-product_id: jbt_core
+:page-product_version: 4.5.2.AM1
+
+== Camel context parameters configurable in properties view for Camel version < 2.18
+
+Before Camel 2.18, the Camel catalog is missing information about Camel Context. Fuse Tooling is now providing this missing piece of information and thus allow to edit Camel Context parameters in Properties view like for any other Camel components. it is activated when there is no element selected on the diagram.
+
+image::./images/camelContextPropertiesView.png[Parameters in Properties view for Camel context]
+
+== Usability improvement: Progress bar when "Switching Camel version"
+
+Since Fuse Tooling 10.1.0, it is possible to switch Camel version. In case the camel version was not cached locally yet and for slow internet connection, this operation can take a while. There is now a progress bar to see the progress.
```

Comment:
"...to switch Camel versions."

Reply:
"...for slow internet connections, ..."

Reply:
Why a 's' here? we are switching to "one" specific version even if it can be done several times with different values.

Reply:
in case you want to use "Camel version" without s then I suggest writing "Switching the Camel version" and even without "" because it is not what the name of the menu item is. Sry, I am also not a native english speaker. So I might be wrong.

Reply:
good point. I modified to use "change" as it is in the UI